src/Components/Space.tsx

```diff
@@ -1,9 +1,10 @@
 import { useEffect, useState, memo } from 'react';
 import { useGame } from '../Contexts/GameContext';
-import { heroColor, Escalator, SpaceTypeName } from '../types';
+import { heroColor, Escalator, SpaceTypeName, DBPlayer, basicAbility, direction } from '../types';
 import { setDoc, getDoc } from '../utils/useFirestore';
 import isEqual from 'lodash/isEqual';
 import { useAudio } from '../Contexts/AudioContext';
+import { async } from '@firebase/util';
```

Comment:
oh tell me about this, i havent used this before

Reply:
omg thats an auto thing. its not even. being used, just automatically came up when i typed async next to a function LOL. its been deleted